test/conftest.py

```diff
@@ -29,7 +29,7 @@ def pytest_generate_tests(metafunc):
 
 
 def get_or_generate_test_cache(update=False, skip=False):
-    test_cache_dir = os.environ.get('QUEST_CACHE_DIR') or os.path.join(quest.util.get_quest_dir(), 'test_cache')
+    test_cache_dir = os.environ.get('QUEST_CACHE_DIR') or os.path.join(quest.util.get_quest_dir(), '.cache/test_cache')
```

Comment:
Again here, instead of `'.cache/test_cache'` do `'.cache', 'test_cache'`.

Reply:
Although maybe it would be better to make it: `or os.path.join(quest.util.get_cache_dir(), '..', 'test_cache')`